popde/density_estimate.py

```diff
@@ -31,6 +31,9 @@ def __init__(self, data, input_transf=None, stdize=False, rescale=None,
             backend : String, Processing method to do KDE calculation
             bandwidth : Float or array of float, bandwidth of kernels used for smoothing
             dim_names : Sequence of dimension names, e.g. ('m1', 'z', 'chi_eff')
+            weights : array_like, optional None (default)
+                      weights of datapoints so  must be the same shape 
+                      as n_samples. If None (default), the samples are assumed to be                        equally weighted 
```

Comment:
```suggestion
            weights : array_like, weights for data points ; if None, the samples are weighted equally
```
We don't need to state the default in the docstring if it is obvious